apps/web/store/archive-queue.ts

```
@@ -0,0 +1,79 @@
+  activeThreadIds: {},
+  totalThreads: 0,
+};
+
+// Create atoms with localStorage persistence for each queue type
+export const queueAtoms = {
+  archive: atomWithStorage("archiveQueue", initialQueueState),
+  delete: atomWithStorage("deleteQueue", initialQueueState),
+  markRead: atomWithStorage("markReadQueue", initialQueueState),
+};
+
+type ActionFunction = (threadId: string, ...args: any[]) => Promise<any>;
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)